### PR TITLE
Refactor CSS, HTML, and JavaScript

### PR DESCRIPTION
**Pull request by jennadilullo8**
**Refactor Iteration 2**

- Driver: Jenna
- Navigator: Liliana & Alex

- Update CSS by deleting empty spaces and placing semi-colons where they are needed.
- Update class of modal to mobileDropDown
- Refactor ternary operation

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -3,34 +3,34 @@
...
 titleInput.addEventListener('keyup', enableSaveButton);
 bodyInput.addEventListener('keyup', enableSaveButton);
+
 //event handlers
 function showFilterStarred() {
-  hamburgerImage.src = (hamburgerImage.src.match("images/menu-close.svg")) ? "images/menu.svg" : "images/menu-close.svg";
-  filterStarred.style.display = (filterStarred.style.display === "block") ? "none" : "block";
-  modal.hidden = (modal.hidden === false) ? true : false;
+  hamburgerImage.src = hamburgerImage.src.match("images/menu-close.svg") ? "images/menu.svg" : "images/menu-close.svg";
+  filterStarred.style.display = filterStarred.style.display === "block" ? "none" : "block";
```
Comment (asodemann18) [inline src/main.js:25]: Nice job refactoring!

File: styles.css [hunk 2]
```diff
@@ -27,21 +21,19 @@ input {
...
-position: fixed; /* Stay in place */
-z-index: 1; /* Sit on top */
-left: 0;
-top: 0;
-width: 100%; /* Full width */
-height: 100%; /* Full height */
-overflow: auto; /* Enable scroll if needed */
-background-color: rgb(0,0,0); /* Fallback color */
-background-color: rgba(100,100,100,0.8); /* Black w/ opacity */
+.mobile-drop-down {
```
Comment (lilianaweimer) [inline styles.css:24]: Hooray for correct indentation!

File: src/main.js [hunk 3]
```diff
@@ -3,34 +3,34 @@
...
 
 function addNewIdea() {
   var titleInputValue = titleInput.value;
   var bodyInputValue = bodyInput.value;
   var newIdea = new Idea(titleInputValue, bodyInputValue);
-
-  if (titleInputValue !== "" && bodyInputValue !== "") {
-    ideaArray.push(newIdea);
-  }
+  ideaArray.push(newIdea);
```
Comment (lilianaweimer) [inline src/main.js:33]: Looks much nicer without the extra if statement :)

**Comment by asodemann18** [comment]
Looks good! Nice update to the naming convention for modal.

**Pull request merged**